pkg/features/kube_features.go

```diff
@@ -645,6 +645,13 @@ const (
 	//
 	// Disables Accelerator Metrics Collected by Kubelet
 	DisableAcceleratorUsageMetrics featuregate.Feature = "DisableAcceleratorUsageMetrics"
+
+	// owner: @arjunrn
```

Comment:
nit: add sig-autoscaling

Reply:
Added @mwielgus and @josephburnett who are part of head and HPA in-charge respectively.